framework/helpers/BaseHtml.php

```diff
@@ -2040,7 +2040,7 @@ public static function getInputName($model, $attribute)
      */
     public static function getInputId($model, $attribute)
     {
-        $name = strtolower(static::getInputName($model, $attribute));
+        $name = mb_strtolower(static::getInputName($model, $attribute));
```

Comment:
Encoding should be passed when using `mb_`  i.e. `mb_strtolower($string, Yii::$app->charset);`.